src/common/schemas/OptionStatement.yml

```diff
@@ -4,6 +4,4 @@
     type:
       $ref: "./NodeType.yml"
     assignment:
-      oneOf:
```

Comment:
This inline `oneOf` was tripping up the codegen, so I split it into a separate file.